src/popsift/common/plane_2d.h

```diff
@@ -285,10 +288,18 @@ template <typename T> class Plane2D : public PitchPlane2D<T>
     }
 
     /** Overwrite the width and height information. Useful if smaller
-     *  planes should be loaded into larger preallocated planes
+     *  planes should be loaded into larger preallocated host planes
+     *  without actually allocating again, but dangerous.
+     *  @warning: pitch is updated (host side)
+     */
+    __host__ void resetDimensionsHost( int w, int h );
+
+    /** Overwrite the width and height information. Useful if smaller
+     *  planes should be loaded into larger preallocated device planes
      *  without actually allocating again, but dangerous.
+     *  @warning: pitch is not updated (device side)
      */
-    __host__ void resetDimensions( int w, int h );
+    __host__ void resetDimensionsDev( int w, int h );
```

Comment:
`resetDimensionsDevice` it's more clear to me

Reply:
Agree, but would require to change existing functions (allocDev, freeDev...) for consistency.

Reply:
Yes, this convention is used in many places. If changed, it should be in another PR.

Reply:
ok i didn't see the others.